test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -3871,77 +3871,6 @@ public void PackCommand_PackProject_PacksFrameworkReferences(string frameworkRef
             }
         }
 
-        [PlatformTheory(Platform.Windows)]
-        [InlineData("Microsoft.WindowsDesktop.App|WindowsForms;Microsoft.WindowsDesktop.App|windowsforms", "net461")]
```

Comment:
Test removed because... it was specific to net461 and no longer needed?

Reply:
No longer needed. 
This was added as an integration test for the E2E scenario. 
The SDK now validates this on their end as well. 

This test failed with other errors, so it doesn't even hit the codepath we want to test anymore.